src/components/app-detail/app-detail.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <v-card v-if="trackDetail.trackName" dark>
+      <v-card-text class="d-flex flex-no-wrap justify-space-between">
+        <div>
+          <v-card-title class="headline white--text">Detail</v-card-title>
+        </div>
+      </v-card-text>
+      <v-container>
+        <v-row dense>
+          <v-col cols="3">
+            <v-avatar class="ma-3" size="125" tile>
+              <img :src="trackDetail.artworkUrl100" />
+            </v-avatar>
+          </v-col>
+          <v-col cols="9">
+            <div>
+              <h2>
```

Comment:
h2 x 5 times? this doesn't have semantic meaning

Reply:
@julianfigueroaGL Probably you could change the div by a section, in this way the h2 could have a semantic sense.

Reply:
done